.github/workflows/CreatePullRequest.yml

```diff
@@ -0,0 +1,39 @@
+name: Create Pull Request
+
+on:
+  pull_request:
+    types: ["opened"]
+
+jobs:
+  assign_and_create_card:
+    name: Assign issue to sender and create Kanban card
+    runs-on: ubuntu-latest
+    # PRs from forks don't have required token authorization
+    if: github.event.pull_request.head.repo.full_name == github.repository
+
+    steps:
+      # https://github.com/actions/github-script
+      - uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const IN_PROGRESS_COLUMN = 4971952;
+            //
+            async function addAssignee(issue, login) {
+                console.log("Assigning to: " + login);
+                await github.issues.addAssignees({
+                    owner: context.repo.owner,
+                    repo: context.repo.repo,
+                    issue_number: issue.number,
+                    assignees: [login]
+                });
+            }
+            //
+            const matches = context.payload.pull_request.body.match(/(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved)\s*#\d+/gi);
+            if(matches) {
+                console.log("Skip, contains '" + matches[0] + "'");
```

Comment:
Log output can be seen here https://github.com/pavel-mikula-sonarsource/GitHubActionPlayground/runs/2003487802?check_suite_focus=true